page_objects/driverfactory.py

```diff
@@ -94,8 +95,11 @@ def run_browserstack(self, os_name, os_version, browser, browser_version,
         if remote_build_name is not None:
             desired_capabilities = self.remote_build_name(desired_capabilities, remote_build_name)
 
-        #Disable screenshots
-        desired_capabilities['browserstack.debug'] = "false"
+        #Screenshot config
+        if screenshot_conf.BS_ENABLE_SCREENSHOTS:
```

Comment:
This can be optimized as - `desired_capabilities['browserstack.debug'] = screenshot_conf.BS_ENABLE_SCREENSHOTS.lower()`

Reply:
Thanks @shivahari Implemented the suggestion